app/code/Magento/Integration/etc/db_schema.xml

```diff
@@ -107,7 +107,7 @@
                 comment="Oauth consumer"/>
         <column xsi:type="timestamp" name="created_at" on_update="false" nullable="false" default="CURRENT_TIMESTAMP"
                 comment="Creation Time"/>
-        <column xsi:type="timestamp" name="updated_at" on_update="true" nullable="false" default="0"
+        <column xsi:type="timestamp" name="updated_at" on_update="true" nullable="false" default="CURRENT_TIMESTAMP"
```

Comment:
Unfortunately, we are not allowed to change column default value according to our [Backward compatible development guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/#db-schema)

Reply:
@sidolov how do you expect me to fix this issue? So there is a real issue with these fields, but due to self proclaimed backwards compatibility this can't be fixed. 

Then we could just remove these fields as well, because they are never going to work anyway if this fix can't be accepted.

Reply:
This is being reviewed internally now, thanks for your help @dverkade!